file.go

```diff
@@ -1,37 +1,115 @@
-package jcsv
+package main
 
-import "os"
+import (
+	"encoding/csv"
+	"fmt"
+	"io/ioutil"
+	"os"
+)
 
 type file struct {
-	// TODO: define variables if needed
+	data []uint8
+	csvData [][]string
+	csvHeader []string
 }
 
 func ParseJsonFile(path string) (file, error) {
-	// TODO: open and read the given file into your `file` object
-	return file{}, nil
+	var f file
+	f.data=nil
+	fileStream,fileOpenError := os.Open(path)
```

Comment:
Please read this for naming conventions.
Note: These are just conventions, not hard rules.

Reply:
You are opening the file but not closing it.